app/components/CreateAssistant.js

```diff
@@ -0,0 +1,84 @@
+'use client';
+import { useState, useContext, useEffect } from 'react';
+import { useFetch } from '@gadgetinc/react';
+import { Button } from '@mui/material';
+import { AppContext } from '../contexts/appDetails';
+
+export default function CreateAssistant() {
+
+  const [instructions, setInstructions] = useState(
+    'Test create Assistant from FE'
```

Comment:
It looks like we are hard coding the instructions here. This is mean to act like a prompt to the AI. So we probably don't want these instructions being passed. We could hard code it in the backend or hardcode it here. 

If we hardcode it here, I think this prompt would be best:

"You are a helpful, friendly assistant. With the document provided, you will help humans answer their questions about this document. You will not stray from the information in this document. If you do not know the answer, you will say so. If there is a request that is outside the context of this document you will inform the human that you can not answer the question. The only exception to this rule, is that the human may ask you fill in a template with some of the information from the other document. You will not stray from the format of this template. If you can't find the information for the template, you will leave the area of the template blank. ";

Reply:
After addressing this comment, this PR LGTM.

Reply:
Take a look at the new prompt. I like the idea of sending the instructions from the FE so we can include the filenames if they are already uploaded. Currently it will not include the file names, but they will be included when I refactor to upload the files before assistant creation.